### DISCUSSION TITLE
Match case how to?

### DISCUSSION THREAD

**Discussion by OnkelTem** [tadashi-aikawa/obsidian-various-complements-plugin#283] [Q&A]
Hi! I discovered this beautiful plugin just today so I'm probably missing something. I want the suggestions to match the case of the letters because currently it seems to be ignoring them. For example, if I type a Russian word like "ответ", I don't want it to suggest "Ответ" starting with а capital letter. How to do this?

![image](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/114060/f81c886e-4afa-44be-9752-7fe3e0f0c0bb)

**Comment by tadashi-aikawa** [comment]
Hi, @OnkelTem :)

This behavior is due to the feature proposed in the following issue. English capitalizes the first letter of a sentence, so even if the dictionary is all lowercase, suggestions for capitalized words are provided.

https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/30

**Reply by OnkelTem** [reply]
It doesn't look like my case tho. 

Currently if I type a lowercase word, the suggestion starts with a capital. If I approve the suggestion, I'll get a capitalized word in the middle of a sentence.

"Это **ответ** на ваш вопрос" > "Это **Ответ** на ваш вопрос".

**Reply by tadashi-aikawa** [reply]
That's so weird

Could you show me the movie? (gif, mp4 etc).

**Reply by OnkelTem** [reply]
Ok, I'll record a video. Sorry for the delay.

Or even better, I'll prepare a test-case vault.
